draft-ietf-taps-interface.md

```diff
@@ -389,10 +389,13 @@ Listener -> ConnectionReceived<Connection>
 // Only receive complete messages in a Conn.Received handler
 Connection.Receive()
 
+messageContext := NewMessageContext()
 Connection -> Received<messageDataRequest, messageContext>
 
 //---- Receive event handler begin ----
-Connection.Send(messageDataResponse)
+messageContext.add(msgPrio, 200)
+messageContext.add(final, true)
+Connection.Send(messageDataResponse, messageContext)
```

Comment:
Where does the messageDataResponse come from. I guess the example does not show everything? But it gets a bit confusing when the messageContext is filled explicitly.

Reply:
sendMessageContext := NewMessageContext()
sendMessageContext.add(msgPrio, 200)
sendMessageContext.add(final, true)
Connection.Send("response", sendMessageContext)